### DISCUSSION TITLE
How to sync or backup user/group/locale information?

### DISCUSSION THREAD

**Discussion by biganan2012** [requarks/wiki#4373] [Help / Questions]
Hi Team,

I am using Git to backup & sync between 2 wikijs instances.  With git, contentx could be synced, while user/group/locale information cannot be synced up. 
Could someone give me a suggestion on it please? Is DB the only option here? Thank u very much.

**Comment by NGPixel** [comment]
Users are stored in the DB and cannot be sync via git.
